### ISSUE TITLE
Add auto PiP on tab switch

### ISSUE THREAD

**Issue by AlexByte** [GoogleChromeLabs/picture-in-picture-chrome-extension#20]
Add the automatic activation of the extension when you leave the tabs, in which the video is played.

**Comment by beaufortfrancois** [comment]
Entering Picture-in-Picture requires a user gesture that can be captured (such as user clicking on the extension icon). I don't think we can do it currently.

However, we are exploring some ways for triggering this.
See Auto Picture-in-Picture: https://developers.google.com/web/updates/2019/02/chrome-73-media-updates#auto-pip

FYI @mounirlamouri

**Comment by AlexByte** [comment]
What is the problem to call a function that activates pip from code?

**Comment by AlexByte** [comment]
Can you remove this check for extensions?

**Comment by AlexByte** [comment]
Add a setting to add this attribute to all videos.

**Comment by AlexByte** [comment]
Add privilege settings for extensions so that you can disable this requirement.

**Comment by AlexByte** [comment]
@beaufortfrancois yes, it's suitable https://chrome.google.com/webstore/detail/always-video-%E1%B4%B4%E1%B4%B7-beta-pip/gcfcmfbcpibcjmcinnimklngkpkkcing

**Comment by quantuminformation** [comment]
pls add this

**Comment by exsesx** [comment]
+1

Arc browser has it, and it's a fantastic feature to have

**Comment by oividioscaeremos** [comment]
+1 yandex browser has it, and it's pretty useful. If I'm changing tabs while watching something, I'd like to keep watching it.

**Comment by shevchenkonik** [comment]
Any chance of this feature being on the roadmap?

**Comment by guidezpl** [comment]
> [@beaufortfrancois](https://github.com/beaufortfrancois) How does this extension works?https://chrome.google.com/webstore/detail/always-video-%E1%B4%B4%E1%B4%B7-beta-pip/gcfcmfbcpibcjmcinnimklngkpkkcing [@robertchan](https://github.com/robertchan)

No longer supported, does anyone have a working alternative?

**Comment by beaufortfrancois** [comment]
AutoPiP now works natively in this extension. See https://github.com/GoogleChromeLabs/picture-in-picture-chrome-extension/pull/93

**Comment by guidezpl** [comment]
Ah thanks. I tested and it works with YouTube, but not with Twitch. Anything we can do?

**Comment by beaufortfrancois** [comment]
The extension works with Twitch as well for me. See

<img width="345" height="214" alt="Image" src="https://github.com/user-attachments/assets/34080908-e64e-4185-b304-6c5de8f8368a" />

**Comment by guidezpl** [comment]
Toggling that off and then back on fixed the issue, thank you!
